release-python-poetry/action.yaml

```diff
@@ -0,0 +1,140 @@
+name: Reusable workflow for building and publishing python packages to TestPyPI or PyPI
+
+on:
+  workflow_call:
+    inputs:
+      scope:
+        description: "Scope of the release; See: https://python-poetry.org/docs/cli/#version"
+        required: true
+        type: string
+      python-version:
+        description: "The python version for setting up poetry."
+        required: false
+        default: 3.7
+        type: number
+      publish-to-test:
```

Comment:
I think the default behavior should be pushing to testpypi so that it is a conscious choice to push to pypi since you cannot delete releases from pypi